lib/bronto/contact.rb

```diff
@@ -20,11 +20,11 @@ def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, includ
       Array.wrap(resp[:return]).map { |hash| new(hash) }
     end
 
-    def self.save(*objs)
+    def self.save(endpoint, *objs)
```

Comment:
As is, the tests will not even run. The required dependencies for testing aren't supported in anything newer than ruby 1.9 (they are requiring debugger which has been replaced with newer things like byebug). 
If we were running 1.9 fairly recently then we may have used these specs. If not, I doubt we ever actually ran them in this gem's context.

I've done an audit of the code on the TJ app side as well and it does not appear that we ever call the class method save, only the instance method, which is designed to hit the correct endpoint based on a save or an update.